### PR TITLE
improvement: ZENKO-483 Monitor CRR Upload doc

### PR DESCRIPTION
**Pull request by bennettbuchanan**

### CODE REVIEW

File: docs/metrics.md [hunk 1]
```diff
@@ -27,16 +27,22 @@ HTTP code.
 ## Routes
 
 Routes are organized in the following fashion:
-`/_/metrics/<extension-type>/<location-name>/<metric-type>`
+`/_/metrics/<extension-type>/<site-name>/<metric-type>/<bucket>/<key>/<version-id>`
```
Comment (rahulreddy) [inline docs/metrics.md]: I recommend keeping `location-name` as we are trying to be consistent with our wording.

**Review by rahulreddy — CHANGES_REQUESTED** [review]
Please change the references of `site` with `location`. This is just to be consistent and avoid people mixing up locations/sites/storage_classes etc.

File: docs/metrics.md (hunk 1 above)
Comment (jonathan-gramain) [reply docs/metrics.md]: Also maybe enclose in `[` and `]` the set of optional bucket, key and version-id fields which depend on the route.

Comment (bennettbuchanan) [reply docs/metrics.md]: Good call, this also needs to be updated to take `version-id` as a query param, reflecting recent changes.

**Review by rahulreddy — APPROVED** [review]

File: docs/metrics.md (hunk 1 above)
Comment (bennettbuchanan) [reply docs/metrics.md]: Updated accordingly.

**Review by jeremyds — APPROVED** [review]

**Pull request merged**